sklearn/cluster/hierarchical.py

```diff
@@ -829,6 +833,17 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         are merged to form node `n_features + i`
     """
 
+    def __init__(self, n_clusters=2, affinity="euclidean",
+                 memory=None,
+                 connectivity=None, compute_full_tree='auto',
+                 linkage='ward', pooling_func=np.mean):
+        super(FeatureAgglomeration, self).__init__(n_clusters=n_clusters,
+                                                   memory=memory,
+                                                   connectivity=connectivity,
+                                                   compute_full_tree=compute_full_tree,
+                                                   linkage=linkage, affinity=affinity,
+                                                   pooling_func=pooling_func)
```

Comment:
Do not pass `pooling_function` to the base class otherwise you'll get a warning in `AgglomerativeClustering.__init__`. This should be fine:

```py
super(FeatureAgglomeration, self).__init__(n_clusters=n_clusters,
    memory=memory,
    connectivity=connectivity,
    compute_full_tree=compute_full_tree,
    linkage=linkage, affinity=affinity)
self.pooling_func=pooling_func
```